test/src/Makefile

```diff
@@ -15,7 +15,7 @@ init:
 ## Run tests
 test: init
 	go mod download
-	go test -v -timeout 60m -run TestExamplesComplete
+	go test -v -timeout 60m
```

Comment:
Do we want `go test -v -timeout 60m  ./...` here to run all tests?

Reply:
Sorry...I was asking if we should include `./...` specifically to include all tests recurively